src/test/java/core/basesyntax/CalculatorTest.java

```diff
@@ -0,0 +1,84 @@
+package core.basesyntax;
+
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class CalculatorTest {
+    private static final double DELTA = 1E-10;
+    private static Calculator calculator;
+
+    @BeforeClass
+    public static void setCalculator() {
+        calculator = new Calculator();
+    }
+
+    @Test
+    public void getSumOk() {
+        double expectedSum = calculator.calculate(2,2,'+');
+        double expectedNegativeSum = calculator.calculate(-2,-3, '+');
+        Assert.assertEquals(4, expectedSum, DELTA);
+        Assert.assertEquals(-5, expectedNegativeSum, DELTA);
+    }
+
+    @Test
+    public void getDivisionOk() {
+        double expectedDivision = calculator.calculate(10,2,'/');
```

Comment:
```suggestion
    public void getDivisionOk() {
        double expectedDivision = calculator.calculate(10, 2, '/');
```
add whitespaces between the parameters in all the places they are needed